hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/ExecutionContext.java

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
nit: license header is not javadoc

```suggestion
/*
```